client/src/commands/UserInputUtil.ts

```diff
@@ -715,6 +715,29 @@ export class UserInputUtil {
         return vscode.window.showQuickPick(quickPickItems, quickPickOptions);
     }
 
+    public static async packageAndInstallQuestion(): Promise<IBlockchainQuickPickItem<boolean>> {
+            const quickPickItems: IBlockchainQuickPickItem<boolean>[] = [
```

Comment:
Can we check the wording with design please?

Reply:
Done!